lib/phoenix_live_view/html.ex

```diff
@@ -0,0 +1,93 @@
+if Code.ensure_loaded?(Phoenix.HTML) do
+  defimpl Phoenix.HTML.FormData, for: Phoenix.LiveView.Socket do
+    def to_form(socket, opts) do
+      params = Keyword.get(opts, :params, %{})
+      name = Keyword.fetch!(opts, :name)|| raise "name expected when using a socket form"
```

Comment:
We probably want to use `:as` here to match phoenix_html

Reply:
Yup!